apis_core/generic/forms/__init__.py

```diff
@@ -93,3 +93,21 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.helper = FormHelper()
         self.helper.add_input(Submit("submit", "Merge"))
+
+
+class GenericEnrichForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        data = kwargs.pop("data", {})
+        instance = kwargs.pop("instance", None)
+        super().__init__(*args, **kwargs)
+        for key, value in data.items():
+            update_key = f"update_{key}"
+            self.fields[update_key] = forms.BooleanField(
+                required=False,
+                label=f"Update {key} from {getattr(instance, key)} to {value}",
+            )
+
+            self.fields[key] = forms.CharField(initial=value, required=False)
```

Comment:
Should the kind of field be copied from the original entities form rather than just using `CharField`? What about when the original form adds some consistency checks? or is a Date?

Reply:
consistency checks should be done on the model, not the form